Sources/MapboxDirections/VisualInstructionComponent.swift

```diff
@@ -147,6 +149,22 @@ public extension VisualInstruction.Component {
             return scale
         }
     }
+
+    struct GuidanceViewImageRepresentation: Equatable {
+
+        /**
+         Initializes an image representation bearing the image at the given base URL.
+         */
+        public init(imageBaseURL: URL?) {
+            self.imageBaseURL = imageBaseURL
+        }
+
+        /**
+         Returns a remote URL to the image file that represents the component.
+         */
+        public let imageBaseURL: URL?
```

Comment:
Unlike the property in ImageRepresentation, this property represents the full URL (minus any authentication-related query parameters), so “base” should be removed from the name.